widgets/mem_freebsd.lua

```diff
@@ -47,14 +47,34 @@ local function worker(format)
     local _swp = { buf = {}, total = nil }
 
     if vm_swap_enabled == 1 and vm_swap_total > 0 then
-        -- Get swap space in bytes
-        _swp.total = vm_swap_total
-        _swp.buf.free = _swp.total - tonumber(vm_stats.v_swapin)
+        -- Initialise variables
+        _swp.usep = 0
+        _swp.inuse = 0
+        _swp.total = 0
+        _swp.buf.free = 0
+
+        -- Read output of swapinfo
+        local f = io.popen("swapinfo")
```

Comment:
I'd suggest forcing swapinfo to reporting in KB, just in case the BLOCKSIZE environment variable is set. Add " -k" to the end of the command, making it:  "swapinfo -k"